### PR TITLE
Added Swift logo with properly width to Readme (SVG from Swift.org)

### PR DESCRIPTION
**Pull request by emilstahl**
As suggested by @krememek in #111

### CODE REVIEW

**Comment by krememek** [comment]
Can you squash this into a single commit?

**Comment by emilstahl** [comment]
I’m obviously an idiot who can’t use git properly.

**Comment by codestergit** [comment]
@emilstahl  Thanks for suggestion. I have reduced the height.

**Comment by emilstahl** [comment]
@krememek Took some trial and error, but finally I managed to squash it into a single commit!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<img src="https://swift.org/assets/images/swift.svg" alt="Swift logo" width="200">
```
Comment (mattbischoff) [inline README.md]: Any reason not to just specify this in Markdown’s image format?

`Swift Logo`

Comment (emilstahl) [inline README.md]: @mattbischoff You can’t set the width in Markdown.

Comment (mattbischoff) [inline README.md]: Ah, got it.

**Comment by gribozavr** [comment]
The readme seems to be using an SVG logo now.

**Pull request closed without merging**